source/vibe/http/client.d

```diff
@@ -599,6 +599,18 @@ final class HTTPClientRequest : HTTPRequest {
 		finalize();
 	}
 
+	/// Writes a multipart upload as a body to the request. This must be written in its entirety because the Content-Length
+	/// must be known beforehand.
+	/// Usage: req.writeBody(new FileMultiPart(req.headers, "Photo", "images/picture.jpg"));
```

Comment:
Needs to be reformatted to be rendered properly in the documentation:
```
/** Writes a multipart form as a body to the request.

	This must be written in its entirety because the Content-Length
	must be known beforehand.

	Usage: `req.writeBody(new FileMultiPart(req.headers, "Photo", "images/picture.jpg"));`
*/
```

Ideal would be to put the example into a documented unit test to verify that it doesn't get out of date.